### PR TITLE
Add Llama/Llama2 support in Question-Answering

### PR DESCRIPTION
**Pull request by kplau1128**
Added Llama/Llama2 support in Question-Answering

### CODE REVIEW

**Comment by kplau1128** [comment]
Fixed missing **LlamaForQuestionAnswering** class issue reported is fixed by applying  [huggingface/transformers](https://github.com/huggingface/transformers/tree/main) [28777](https://github.com/huggingface/transformers/pull/28777).

Fixed _**run_qa.py**_ script to support Llama tokenizer.

**Comment by regisss** [comment]
`LlamaForQuestionAnswering` was added in Transformers v4.38, which will be supported by Optimum Habana in a few weeks. Let's wait for that before continuing this PR.

**Comment by kplau1128** [comment]
Verified with updated transformer v4.38.2 #788, work fine.

File: examples/question-answering/run_qa.py [hunk 1]
```diff
@@ -377,6 +377,10 @@ def main():
         token=model_args.token,
         trust_remote_code=model_args.trust_remote_code,
     )
+    if "Llama" in model_args.model_name_or_path:
```
Comment (regisss) [inline examples/question-answering/run_qa.py:380]:
```suggestion
    if config.model_type == "llama":
```

**Review by regisss — COMMENTED** [review]
Can you update the main branch of your fork and rebase this PR on it please?

File: examples/question-answering/run_qa.py (hunk 1 above)
Comment (kplau1128) [reply examples/question-answering/run_qa.py]: fixed

**Comment by kplau1128** [comment]
rebased

**Review by regisss — APPROVED** [review]
LGTM!

I pushed two additional commits to refine a bit the README and run `make style`.

**Pull request merged**